presto-raptor/src/main/java/com/facebook/presto/raptor/storage/organization/ShardCompactor.java

```diff
@@ -95,7 +96,8 @@ public ShardCompactor(StorageManager storageManager, ReaderAttributes readerAttr
             throws IOException
     {
         for (UUID uuid : uuids) {
-            try (ConnectorPageSource pageSource = storageManager.getPageSource(FileSystemContext.DEFAULT_RAPTOR_CONTEXT, uuid, bucketNumber, columnIds, columnTypes, TupleDomain.all(), readerAttributes)) {
+            // todo
```

Comment:
what is this todo?

Reply:
So in this commit, we didn't touch the functionality in compactor, but due to interface changing, this part also needs change.
As suggested, I left a todo, which I fix in my next commit.
We will be merging several of the commits into bigger commits later, this will disappear after reorganize.